src/transformers/modeling_t5.py

```diff
@@ -27,7 +27,7 @@
 from torch.nn import CrossEntropyLoss
 
 from .configuration_t5 import T5Config
-from .file_utils import DUMMY_INPUTS, DUMMY_MASK, add_start_docstrings
+from .file_utils import DUMMY_INPUTS, DUMMY_MASK, add_start_docstrings, add_start_docstrings_to_callable
```

Comment:
In general I changed the docstring to "Bert"-style docstring since Bert seems to have the cleanest docstrings. @thomwolf